src/system/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Box } from '../Box';
+
+export interface SkeletonProps {
+	variant?: string;
+	width?: number | string;
+	height?: number | string;
+	borderRadius?: number;
+}
+
+export const Skeleton = ( {
+	variant = 'text',
+	width = '100%',
+	height = '30px',
+	borderRadius = 1,
+	times = 1,
+	...props
+} ) => (
+	<>
+		{ Array.from( { length: times } ).map( ( i, index ) => (
+			<Box
+				key={ index }
+				sx={ {
+					borderRadius: variant === 'circle' ? '50%' : borderRadius,
+					width,
+					height,
+					backgroundColor: 'skeleton.background',
+					animation: 'pulse 1.5s ease-in-out infinite',
+					'@keyframes pulse': {
+						'0%': {
+							opacity: 0.1,
+						},
+						'50%': {
```

Comment:
Do we need to increase opacity for a11y too??

Reply:
@rebeccahum Since it's purely decorative and not the only way to communicate progress, contrast is not strictly required. The idea is to keep with the `<Spinner />` component, and have the new component only to be a placeholder preview of content before the data gets loaded.